demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -181,8 +181,18 @@ export class DemoMeetingApp {
     }
   }
 
-  getAudioInputDeviceID = (): string => {
+  getAudioInputDevice = (): string | MediaStream => {
     const dropdownList = document.getElementById('audio-input') as HTMLSelectElement;
+    if(dropdownList.value == 'No Audio') {
```

Comment:
Where is this drop down value from? 
Also

Reply:
This value is selected from the mic drop down inside MeetingReadinessChecker demo browser app.
<img width="789" alt="MeetingReadinessChecker mic dropdown" src="https://user-images.githubusercontent.com/123211697/217442237-288d6fa6-bd91-4e73-81d0-4b1294fb9385.png">

Reply:
Can you point me to the code where its added? Its not there in my local repo.

Reply:
L500-L505 of [demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts]([demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts](https://github.com/aws/amazon-chime-sdk-js/pull/2561/commits/6b747db1a72684c799b52afb1e4fa65d25ffe9e3#diff-d37b035014cecb05ff1a11a4ea1543eb5be17eb6e249101b484744ecd08c4909)) file